modulus/datapipes/gnn/ahmed_body_dataset.py

```diff
@@ -92,7 +108,7 @@ def __init__(
         data_dir: str,
         split: str = "train",
         num_samples: int = 10,
-        invar_keys: List[str] = [
+        invar_keys: Iterable[str] = (
```

Comment:
Does this change the docstrings here: https://github.com/NVIDIA/modulus/pull/387/files#diff-1af1f52943601c7e922fb5b8b6fde91bfa932071db5abcdcfdc8c37e759ca7adR88 ? Or do we want to keep it the same indicating `List[str]` as the preferable option?

Reply:
Good point - fixed the docstring. The code works with any iterable equally well, so there is no need to use more specific type.